src/challenges/scramblies.js

```diff
@@ -0,0 +1,30 @@
+const scramble = (str1, str2) => {
+  let str1_character_count = {}
+  let str2_character_count = {}
+  for (character of str1) {
+    if (character in str1_character_count) {
+      str1_character_count[character] += 1
+    } else {
+      str1_character_count[character] = 1
+    }
+  }
+  for (character of str2) {
+    if (character in str2_character_count) {
+      str2_character_count[character] += 1
+    } else {
+      str2_character_count[character] = 1
+    }
+  }
+  console.log(str1_character_count, str2_character_count)
+  for (key in str2_character_count) {
+    if (str1_character_count[key] === undefined) {
+      return false
+    } else if (str1_character_count[key] < str2_character_count[key]) {
+      return false
+    }
+  }
```

Comment:
What if str1 has an additional character?

Reply:
This is the downside of not having the requirements available in the Pull request.  I think moving forward you should fill out the comments of pull request with what the code is supposed to do.

Reply:
okay. part of fixing up the read me was going to be adding the challenges information. i will just start doing that going forward